jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/saved/SavedFiltersPresenter.java

```diff
@@ -55,7 +55,7 @@ public String getTitle() {
 
     @PostConstruct
     public void init() {
-        loadSavedFilters();
+        filterSettingsManager.getDefaultActiveFilterKey(defaultFilter -> loadSavedFilters(defaultFilter));
```

Comment:
in here I think default filter is an attribute of the filter itself. You should continue using just _loadSavedFilters_ and that would set the default filter automatically.